app/models/rbac.rb

```diff
@@ -222,7 +222,7 @@ def self.find_targets_filtered_by_parent_ids(parent_class, scope, find_options,
   end
 
   def self.find_targets_filtered_by_ids(scope, find_options, u_filtered_ids, filtered_ids)
-    total_count  = total_scope(scope, u_filtered_ids, find_options[:conditions], find_options[:include]).count
+    total_count  = total_scope(scope, u_filtered_ids, find_options[:conditions], find_options[:include]).count(:all)
```

Comment:
@chrisarcand @jrafanie The reason it is there is if a `.select` is present with `virtual_attribute` columns, postgres will blow up because it will try making a query like this:

```SQL
SELECT COUNT(*, aggregate_something_something, aggregate_foo_bar) FROM ...
```

Which is bad.  Where this was introduced, and if this is the right place for it to be introduced for before the refactoring happened is what I don't know.